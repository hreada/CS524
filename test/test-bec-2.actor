letactor(
  bind(a1, lambda(m, seq(become(0), 1))),
  send(a1, nil))
