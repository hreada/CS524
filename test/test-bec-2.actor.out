<T>
  <actors>
    <actor>
      <k> exec ( 1 ) </k>
      <id> _6 </id>
    </actor>
    <actor>
      <k> ready ( 0 ) </k>
      <id> _1 </id>
    </actor>
    <actor>
      <k> exec ( nil ) </k>
      <id> initactor </id>
    </actor>
  </actors>
  <messages> .MessageCellBag </messages>
  <definedaddr> ListItem ( initactor ) ListItem ( _1 ) ListItem ( _6 ) </definedaddr>
</T>
